README.md

```diff
@@ -1,2 +1,19 @@
 # terraform-aws-lambda-cloudwatch-alarms
 Terraform module for creating a set of Lambda alarms and outputting to an endpoint
```

Comment:
Terraform module for creating a set of Lambda alarms and outputting to an SNS endpoint